src/rkopenmdao/error_controller.py

```diff
@@ -94,8 +94,10 @@ def _estimate_next_step_function(self, current_norm, current_time):
         delta_time_new *= (self.tol / current_norm) ** self.alpha
         delta_time_new *= (self.local_error_norms[0] / self.tol) ** self.beta
         delta_time_new *= (self.tol / self.local_error_norms[1]) ** self.gamma
-        delta_time_new *= (current_time / self.delta_time_steps[0]) ** self.a
-        delta_time_new *= (self.delta_time_steps[0] / self.delta_time_steps[1]) ** self.b
+        if self._delta_time_steps[0]:
```

Comment:
Better use `if self._delta_time_steps[0] is not None.` here (and equivalently two lines below. That way, we can differentiate between `None` and `0.0`.